cmake/scripts/check_python_package.py

```diff
@@ -1,4 +1,7 @@
-# file which is invoked by the cmake build system to check if all necessary python packages are installed.
+#!/usr/bin/env python3
```

Comment:
This is also missing for `configure_cmake_for_target.py`, in the same `cmake/scripts` directory. Would you like to update it too?